### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by domnikl**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/Xkd11q).

### CODE REVIEW

File: Behavioral/Iterator/BookList.php [hunk 1]
```diff
@@ -13,7 +12,7 @@ public function getBook($bookNumberToGet)
             return $this->books[$bookNumberToGet];
         }
 
-        return null;
+        return;
```
Comment (sergeyklay) [inline Behavioral/Iterator/BookList.php]: null !== void

File: Behavioral/Memento/Tests/MementoTest.php [hunk 2]
```diff
@@ -125,40 +124,39 @@ public function testStateWithDifferentObjects()
...
+        $this->assertAttributeEquals($first, 'state', $originator);
 
         $first_snapshot = $originator->getStateAsMemento();
-        $this->assertAttributeEquals($first, "state", $first_snapshot);
+        $this->assertAttributeEquals($first, 'state', $first_snapshot);
 
-        $second       = new \stdClass();
-        $second->data = "bar";
+        $second = new \stdClass();
+        $second->data = 'bar';
```
Comment (sergeyklay) [inline Behavioral/Memento/Tests/MementoTest.php]: :-1:

File: Behavioral/NullObject/Service.php [hunk 3]
```diff
@@ -28,7 +28,7 @@ public function __construct(LoggerInterface $log)
     public function doSomething()
     {
         // no more check "if (!is_null($this->logger))..." with the NullObject pattern
-        $this->logger->log('We are in ' . __METHOD__);
+        $this->logger->log('We are in '.__METHOD__);
```
Comment (sergeyklay) [inline Behavioral/NullObject/Service.php]: :-1:

File: Behavioral/Observer/Tests/ObserverTest.php [hunk 4]
```diff
@@ -2,15 +2,14 @@
 
 namespace DesignPatterns\Behavioral\Observer\Tests;
 
-use DesignPatterns\Behavioral\Observer\UserObserver;
 use DesignPatterns\Behavioral\Observer\User;
+use DesignPatterns\Behavioral\Observer\UserObserver;
```
Comment (sergeyklay) [inline Behavioral/Observer/Tests/ObserverTest.php]: ???

File: Behavioral/Iterator/BookList.php (hunk 1 above)
Comment (Koc) [inline Behavioral/Iterator/BookList.php]: does related RFC was accepted?

Comment (domnikl) [inline Behavioral/Iterator/BookList.php]: in PHP, this is equal, as there is null returned when no explicit value was given.

File: Behavioral/Memento/Tests/MementoTest.php (hunk 2 above)
Comment (domnikl) [inline Behavioral/Memento/Tests/MementoTest.php]: ?

File: Behavioral/Iterator/BookList.php (hunk 1 above)
Comment (Koc) [inline Behavioral/Iterator/BookList.php]: @domnikl https://wiki.php.net/rfc/void_return_type heh, yes, RFC was accepted

File: Behavioral/Memento/Tests/MementoTest.php (hunk 2 above)
Comment (sergeyklay) [inline Behavioral/Memento/Tests/MementoTest.php]: with alignment was more human-readable

Comment (rob006) [inline Behavioral/Memento/Tests/MementoTest.php]: https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md#inter-line-alignment

Comment (sergeyklay) [inline Behavioral/Memento/Tests/MementoTest.php]: Hmm... reasonably

**Pull request merged**